File: PROG/Swift/Level1/12901.swift
func solution(_ a:Int, _ b:Int) -> String {
    let dayOfWeek = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"]
    let monthOfDays = [5, 1, 2, 5, 0, 3, 5, 1, 4, 6, 2, 4]
    let first = (b + monthOfDays[a - 1] - 1) % 7
    
    return dayOfWeek[first]
}

File: PROG/JS/LEVEL-1/12901.js
function solution(a, b) {
  var endDayOfMonth = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
  var firstDayOfMonth = ["FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"];
  var tempVar = b - 1; // 기준일이 1일, -1을 해준다.
  for (var i = 0; i < a - 1; i++) {
    tempVar += endDayOfMonth[i];
  }
  return firstDayOfMonth[tempVar % 7];
}
